components/causal_ts/src/config.rs

```diff
@@ -38,6 +45,7 @@ impl Default for Config {
         Self {
             renew_interval: ReadableDuration::millis(crate::tso::TSO_BATCH_RENEW_INTERVAL_DEFAULT),
             renew_batch_min_size: crate::tso::TSO_BATCH_MIN_SIZE_DEFAULT,
+            batch_list_capacity: crate::tso::TSO_BATCH_LIST_CAPACITY,
```

Comment:
TSO_BATCH_LIST_CAPACITY
```suggestion
            batch_list_capacity: crate::tso::DEFAULT_TSO_BATCH_LIST_CAPACITY,
```